mvp-spec/audit-log-requirements.md

```diff
@@ -181,7 +181,7 @@ OpenRTB requests and the election of a bid winner:
 
 <!--partial-begin { "files": [ "transaction-nominal-case.mmd" ], "transformation": "mermaid" } -->
 <!-- ⚠️ GENERATED CONTENT - DO NOT MODIFY DIRECTLY ⚠️ -->
-![generated-transaction-nominal-case](assets\generated-transaction-nominal-case.svg)
+![generated-transaction-nominal-case](assets/generated-transaction-nominal-case.svg)
```

Comment:
"/" on mac and "\" on Windows? I guess we should be constant on this. Example: always "/". However, I didn't see a fix regarding that in this commit. If so, you can fix it in the code or dismiss this line.

Reply:
path is "clever" enough to use the "right" delimiter, so in your case, it used `\`, which is concerning.
I will find a way to use `/` everywhere.

Reply:
This is indeed what I thought.
I guess we can provide a function helper in the file.ts that rework a path and use it in the interpretor.ts. Up to you. :-)